health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/subBoundaryView.js

```
@@ -7,34 +7,32 @@ const SubBoundaryView = ({ title, arr }) => {
-            ): (
-                null
-            )
-        }
+                arr && arr.length > 0 ? (
+                    <Card>
+                        <HeaderComp title={title} />
+                        {/* Flex container for the chips */}
+                        <div className="subBoundarycomp-container">
+                            {arr.map((el, ind) => {
+                                return (
+                                    <div className="digit-tag-container">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)